scripts/apps/authoring/authoring/directives/AuthoringDirective.ts

```diff
@@ -564,6 +565,13 @@ export function AuthoringDirective(
              * in $scope.
              */
             $scope.publish = function() {
+                if ($scope.fieldsWithInvalidChar.length > 0) {
```

Comment:
Can you handle publishing validation on the backend?

Reply:
And if we keep it like the way it is we can't perform validations in the backend as we don't know which field to validate i.e. which field has this character validation on.

Reply:
That's a good point. Let's then add a checkbox to the content profile, whether the field has to be validated or not and keep the validation pattern common. You can put the pattern in the server configs and it will get downloaded to the client.